packages/frontend/src/creation-form/components/outcomes-configuration/index.tsx

```diff
@@ -2,71 +2,69 @@ import { BigNumber, utils } from "ethers";
     t,
     templates,
-    outcomesData,
+    state,
+    onStateChange,
     onNext,
 }: OutcomesConfigurationProps): ReactElement => {
-    const [data, setData] = useState<RawOutcomeDataMap>(
-        outcomesData.reduce((accumulator: RawOutcomeDataMap, data, i) => {
-            const binary =
-                data.lowerBound.eq(BigNumber.from(0)) &&
-                data.higherBound.eq(BigNumber.from(1));
+    // const [data, setData] = useState<RawOutcomeDataMap>(
+    //     outcomesData.reduce((accumulator: RawOutcomeDataMap, data, i) => {
+    //         const binary =
+    //             data.lowerBound.eq(BigNumber.from(0)) &&
+    //             data.higherBound.eq(BigNumber.from(1));
 
-            const lowerBoundValue = binary
-                ? "0"
-                : utils.formatUnits(data.lowerBound.toString(), 18);
-            const lowerBoundFormattedValue =
-                binary || !lowerBoundValue
-                    ? ""
-                    : utils.commify(lowerBoundValue);
+    //         const lowerBoundValue = binary
+    //             ? "0"
+    //             : utils.formatUnits(data.lowerBound.toString(), 18);
+    //         const lowerBoundFormattedValue =
+    //             binary || !lowerBoundValue
+    //                 ? ""
+    //                 : utils.commify(lowerBoundValue);
 
-            const higherBoundValue = binary
-                ? "0"
-                : utils.formatUnits(data.higherBound.toString(), 18);
-            const higherBoundFormattedValue =
-                binary || !higherBoundValue
-                    ? ""
-                    : utils.commify(higherBoundValue);
-            accumulator[templates[i].id] = {
-                binary,
-                lowerBound: {
-                    value: lowerBoundValue,
-                    formattedValue: lowerBoundFormattedValue,
-                },
-                higherBound: {
-                    value: higherBoundValue,
-                    formattedValue: higherBoundFormattedValue,
-                },
-            };
-            return accumulator;
-        }, {})
-    );
+    //         const higherBoundValue = binary
+    //             ? "0"
+    //             : utils.formatUnits(data.higherBound.toString(), 18);
+    //         const higherBoundFormattedValue =
+    //             binary || !higherBoundValue
+    //                 ? ""
+    //                 : utils.commify(higherBoundValue);
+    //         accumulator[templates[i].id] = {
+    //             binary,
+    //             lowerBound: {
+    //                 value: lowerBoundValue,
+    //                 formattedValue: lowerBoundFormattedValue,
+    //             },
+    //             higherBound: {
+    //                 value: higherBoundValue,
+    //                 formattedValue: higherBoundFormattedValue,
+    //             },
+    //         };
+    //         return accumulator;
+    //     }, {})
+    // );
```

Comment:
why is this commented code? should we keep it?

Reply:
Whoops, my bad. Addressed in 76663bf93e9c33bb02ed88672e39720c7785e85b